content/develop/whats-new/8-2.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Redis 8.2
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+- rc
+description: What's new in Redis 8.2
+linkTitle: What's new in Redis 8.2
+weight: 4
+---
+
+Redis 8.2 builds on the foundation of Redis 8.0 with significant performance and memory optimizations, enhanced streaming capabilities, and improved cluster management tools.
+
+This release delivers major improvements across multiple areas:
+- Enhanced Redis Streams with new commands for better consumer group management
+- New bitmap operations for advanced bit manipulation
+- Advanced cluster monitoring with per-slot usage metrics
+- Optimized vector search with new compression algorithms
+- Enhanced keyspace notifications with new event types
+- More than 15 performance and resource utilization improvements
+
+Below is a detailed breakdown of these updates.
+
+## New features
+
+### Enhanced Redis Streams
+
+Redis 8.2 introduces powerful new commands that simplify consumer group management and stream lifecycle operations:
+
+- [`XDELEX`]({{< relref "/commands/xdelex/" >}}): Delete entries from streams with advanced consumer group handling options.
+- [`XACKDEL`]({{< relref "/commands/xackdel/" >}}): Acknowledge and delete stream entries in a single atomic operation.
+
+These commands include `KEEPREF`, `DELREF`, and `ACKED` options that give you precise control over how consumer group references are handled during entry deletion. This simplifies common patterns where you need to clean up acknowledged entries while maintaining consumer group integrity.
+
+The [`XADD`]({{< relref "/commands/xadd/" >}}) and [`XTRIM`]({{< relref "/commands/xtrim/" >}}) commands have also been extended with these same options, providing consistent behavior across all stream management operations.
+
+### Cluster monitoring improvements
+
+The new [`CLUSTER SLOT-STATS`]({{< relref "/commands/cluster-slot-stats/" >}}) command provides detailed per-slot usage metrics including:
+- Key count per slot
+- CPU time consumption
+- Network I/O statistics
+
+This enhanced visibility helps you optimize cluster performance and identify hotspots in your data distribution.
+
+### Vector search enhancements
+
+Redis 8.2 introduces the SVS-VAMANA vector index type, which supports vector compression and is optimized for Intel machines. This new index type can significantly reduce memory usage while maintaining search performance for vector similarity operations.
```

Comment:
Please remove this paragraph.
We will have to remove this from OSS on 8.2.1 because it uses a intel library that is not compatible with AGPL.